mlir/utils/performance/tuningRunner.py

```diff
@@ -263,7 +263,7 @@ def main(args=None):
     parser.add_argument(
         '--data-type',
          nargs='+',
-         choices=["f32", "f16", "i8"],
+         choices=["f32", "f16", "i8", "f32_f32", "f16_f16", "i8_i32", "i8_i8"],
```

Comment:
I don't think we need the 'f32_f32` ones.

Reply:
Yes, I would say that we only need "i8_i8" and "i8_i32". We can assume that the others mean same input/output type.